### PR TITLE
Update debug to have timestamps

### PR DESCRIPTION
**Pull request by waveywaves**
**What this PR does / why we need it**:
This PR adds timestamp to debug statements
fixes https://github.com/helm/helm/issues/6336

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

**Comment by waveywaves** [comment]
cc @bacongobbler Can you take a look at this.
@freeseacher What do you think about this. Should I make any more changes ?

File: cmd/helm/helm.go [hunk 1]
```diff
@@ -55,7 +56,8 @@ func init() {
 
 func debug(format string, v ...interface{}) {
 	if settings.Debug {
-		format = fmt.Sprintf("[debug] %s\n", format)
+		timeNow := time.Now().UTC().String()
```
Comment (thomastaylor312) [inline cmd/helm/helm.go]: This should probably use something like RFC3339 formatted timestamps instead of just the raw string implementation

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
I wanted to have a discussion around whether or not timestamps would be valuable here or not. Because this is purely for the CLI, I am not entirely sure they will be useful because each command is relatively short-lived. I can see the argument for having it in the logging that comes with `--wait` though, so I am completely open for discussion on this

File: cmd/helm/helm.go (hunk 1 above)
Comment (thomastaylor312) [reply cmd/helm/helm.go]: But let me check on the time format before doing anything here

Comment (thomastaylor312) [reply cmd/helm/helm.go]: I think I'd go with RFC3339, unless you have inclinations otherwise

**Comment by thomastaylor312** [comment]
@waveywaves As I mentioned in my comment, I don't think adding timestamps is really necessary seeing as this is a function used purely by the CLI. Anyone who uses it as a library can attach their own logger with timestamps and if you are using it in CI or other automated usage, timestamps are automatically attached. Please let me know if there is a good argument for them I am missing

**Comment by freeseacher** [comment]
>timestamps are automatically attached
sorry, but they are not. gitlab does not provide such functionality :( 
probably it is corner case and timestpamping should be added with flags

**Comment by bacongobbler** [comment]
Thanks @waveywaves for the contribution. Right now the core maintainers are trying to focus on bug fixes so we can release Helm 3.0 in a timely manner. Can we re-discuss this feature after Helm 3.0 has been released?

**Comment by waveywaves** [comment]
@bacongobbler @thomastaylor312 From my point of view as well, this does not seem to be something that is extremely critical. I have only updated this wrt the feature proposal by @freesearcher.

**Comment by bacongobbler** [comment]
Apologies for the automatic closure. We just switched the `dev-v3` branch to `master`, and the old `master` branch is now `dev-v2`. Feel free to re-open a new PR with this change. Thanks again and sorry for the hassle!

**Comment by freeseacher** [comment]
please reopen it. still usefull in my opinion

**Comment by bacongobbler** [comment]
@freeseacher once the base branch has been closed, the PR cannot be re-opened. That's why I'm asking @waveywaves to open a new PR.

This was a clerical error on our end as part of the migration. Sorry about that.

**Pull request closed without merging**